doc/user/content/manage/access-control/_index.md

```diff
@@ -10,116 +10,298 @@ menu:
+roles in a Materialize region, you can use the [`ALTER DEFAULT PRIVILEGES`](/sql/alter-default-privileges/)
+command.
 
-Inheritance in RBAC allows you to create roles that inherit privileges from
-other roles. Inheritance only applies to role privileges. Role attributes are
-not inherited. Inheriting privileges allows you to minimize the number of roles you have to manage.
+```sql
+# Use SHOW ROLES to list existing roles in the system, which are 1:1 with invited users
+SHOW ROLES;
+
+-- Example: grant read-only access to all object types, to all roles in the
+-- system via the PUBLIC role
+--For PostgreSQL compatibility reasons, TABLE is the object type for sources,
+--views, and materialized views
+ALTER DEFAULT PRIVILEGES FOR ALL ROLES GRANT SELECT ON TABLES TO PUBLIC;
+ALTER DEFAULT PRIVILEGES FOR ALL ROLES GRANT USAGE ON DATABASES TO PUBLIC;
+ALTER DEFAULT PRIVILEGES FOR ALL ROLES GRANT USAGE ON SCHEMAS TO PUBLIC;
+```
+
+#### Inviting users as administrators
+
+The path of least resistance to setting up RBAC is...not setting it up. If you
+invite all users as `Organization Admin`, all users will have the _superuser_
+role.
+
+Keep in mind that you will be giving all users unrestricted access to all
+resources in your Materialize region, as well as all features in the
+Materialize console. While this might be a viable solution when trialing
+Materialize with a small number of users, it can be **hard to revert** once you
+decide to roll out a RBAC strategy for your Materialize organization.
+
+As an alternative, you can approximate the set of privileges of a _superuser_ by
+instead modifying the default privileges to be wildly permissive:
+
+```sql
+-- Use SHOW ROLES to list existing roles in the system, which are 1:1 with invited users
+SHOW ROLES;
+
+-- Example: approximate full-blown admin access by modifying the default
+-- privileges inherited by all roles via the PUBLIC role
+GRANT USAGE, CREATE ON SCHEMA materialize.public TO PUBLIC;
+GRANT ALL PRIVILEGES ON DATABASE materialize TO PUBLIC;
+GRANT USAGE, CREATE ON CLUSTER default TO PUBLIC;
+GRANT CREATEDB, CREATECLUSTER ON SYSTEM TO PUBLIC;
+GRANT CREATEROLE ON SYSTEM TO PUBLIC;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA materialize.public TO PUBLIC;
+GRANT ALL PRIVILEGES ON ALL TYPES IN SCHEMA materialize.public TO PUBLIC;
+GRANT ALL PRIVILEGES ON ALL SECRETS IN SCHEMA materialize.public TO PUBLIC;
+GRANT ALL PRIVILEGES ON ALL CONNECTIONS IN SCHEMA materialize.public TO PUBLIC;
+
+ALTER DEFAULT PRIVILEGES FOR ALL ROLES IN SCHEMA materialize.public GRANT ALL PRIVILEGES ON TABLES TO PUBLIC;
+ALTER DEFAULT PRIVILEGES FOR ALL ROLES IN SCHEMA materialize.public GRANT ALL PRIVILEGES ON TYPES TO PUBLIC;
+ALTER DEFAULT PRIVILEGES FOR ALL ROLES IN SCHEMA materialize.public GRANT ALL PRIVILEGES ON SECRETS TO PUBLIC;
+ALTER DEFAULT PRIVILEGES FOR ALL ROLES IN SCHEMA materialize.public GRANT ALL PRIVILEGES ON CONNECTIONS TO PUBLIC;
+```
```

Comment:
Can we include a note about why we're restricting the privileges to the `materialize.public` schema.

Reply:
Can you remind me again why that is? I'll address that in a follow-up PR.